<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Only change those fields if you know what to do!</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>10</y>
     <width>340</width>
     <height>241</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>340</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="General">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>80</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>330</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Offline data</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path to offline data:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="offlineDataField">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="offlineDataFolderBrowse">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="offlineData_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Use offline data</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Default path to the .epw output folder</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="outputPathField">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="outputhPathFolderBrowse">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FTP">
    <attribute name="title">
     <string>FTP</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adress to the FTP server</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ftpAdressField">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ftpUserField">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ftpPassField">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Path to the hourly data</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ftpPathField">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ftpDefaultsButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset to default</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Data">
    <attribute name="title">
     <string>Data</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimal number of records for the set bo be accepted</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="minRecField">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(max 8760)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="tabular_checkBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Save reports in tabular form</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="interpolate_data_checkBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Interpolate missing data</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>offlineDataFolderBrowse</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>offlineDataFolderBrowse_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputhPathFolderBrowse</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>outputPathFieldBrowse_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ftpDefaultsButton</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>ftpDefaultsButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>offlineDataFolderBrowse_clicked()</slot>
  <slot>outputPathFieldBrowse_clicked()</slot>
  <slot>ftpDefaultsButton_clicked()</slot>
 </slots>
</ui>
